vite.config.mjs

```
@@ -37,16 +37,16 @@ export default defineConfig({
-        // Serve markdown files - handle both /src/content/blog and /ThomasJButler/src/content/blog paths
+        // Serve markdown files from /src/content/blog path
         server.middlewares.use((req, res, next) => {
           // Check if this is a request for a blog markdown file
           const url = req.url;
           let blogPath = null;
 
           if (url.startsWith('/src/content/blog/')) {
             blogPath = url.replace('/src/content/blog/', '');
-          } else if (url.startsWith('/ThomasJButler/src/content/blog/')) {
-            blogPath = url.replace('/ThomasJButler/src/content/blog/', '');
+          } else if (url.startsWith('/src/content/blog/')) {
```

Biome (2.1.2)

[error] This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain.



(lint/suspicious/noDuplicateElseIf)